docs/reference/createPlugin.md

```diff
@@ -17,21 +17,24 @@ type PluginHooks = {
 };
 ```
 
-[Read more about the router here](createPlugin-router.md)
+- [Read more about the router here](createPlugin-router.md)
+- [Read more about feature flags here](createPlugin-feature-flags.md)
 
 ## Example Uses
 
 ### Creating a basic plugin
 
-Showcasing adding multiple routes and a redirect.
+Showcasing adding multiple routes, a feature flag and a redirect.
 
 ```jsx
 import { createPlugin } from '@backstage/core';
 import ExampleComponent from './components/ExampleComponent';
 
 export default createPlugin({
   id: 'new-plugin',
-  register({ router }) {
+  register({ router, featureFlags }) {
+    featureFlags.registerFeatureFlag('enable-example-component');
```

Comment:
I think `featureFlags.flag` would look nicer, but then again I think we should switch to `router.route` too, since everything we're doing here is registration :grin:. Fine with either and will bring it up for discussion later.

Reply:
`featureFlags.flag` sounds pretty confusing to me. Doesn't signal at all what it does. I can get behind `featureFlags.register` though.

Reply:
Yeah I know, would be fine with `register` as well. Kinda depends on docs and what other things we add look like.

Reply:
Changed to `register`